### PR TITLE
Adding add domain endpoint to be able to add more than one domain to an account

### LINKED ISSUE #133: create an handler to add a domain to a specific account
It should be possible to link a domain to an account.

We should have the following endpoint:
* `POST /api/v1/domainassociation/customers/{customerIdentifier}/domains`
The **Content-Type** header should be set to **application/json** and the Body should look like:
```
{
    "domainName": "example.com"
}

No payload is expected as a response. AppMarket will use the HTTP code to determine if the operation was successful or not. The expected HTTP code for a success is `200`.

### PR DESCRIPTION
**Pull request by davidchau**
It should be possible to link a domain to an account.

We should have the following endpoint:
* `POST /api/v1/domainassociation/customers/{customerIdentifier}/domains`
The **Content-Type** header should be set to **application/json** and the Body should look like:
```
{
    "domainName": "example.com"
}
```
No payload is expected as a response. AppMarket will use the HTTP code to determine if the operation was successful or not. The expected HTTP code for a success is `200`.

resolves #133

### CODE REVIEW

File: src/test/java/com/appdirect/sdk/feature/CanVerifyDomainsWhenModuleImportedIntegrationTest.java [hunk 1]
```diff
@@ -127,6 +137,21 @@ public void testValidateOwnership() throws Exception {
 		assertThat(fakeAppmarket.domainVerificationStatuses().get(0).isVerified()).isTrue();
 	}
 
+	@Test
+	public void testAddDomain() throws Exception {
+		//Given
```
Comment (kais-ad) [inline src/test/java/com/appdirect/sdk/feature/CanVerifyDomainsWhenModuleImportedIntegrationTest.java]: Remove this comment, or add the //Execute and //Assert

Comment (davidchau) [reply src/test/java/com/appdirect/sdk/feature/CanVerifyDomainsWhenModuleImportedIntegrationTest.java]: will add the `// Then` in the next PR

**Comment by hooriak** [comment]
LGTM.

**Comment by hooriak** [comment]
merging..

**Pull request merged**